src/targets/gpu/include/migraphx/gpu/convolution.hpp

```diff
@@ -59,6 +59,7 @@ struct miopen_convolution
 {
     Op op;
     bool int8_x4_format               = false;
+    bool tune                         = false;
```

Comment:
This should go into the `context` and not the operator.

Reply:
Agree. Added to context. but for that, i  had to add additional method in `context` virtual class.